config/default.yml

```diff
@@ -5593,6 +5593,12 @@ Style/Strip:
   Enabled: true
   VersionAdded: '0.36'
 
+Style/RedundantStructKeywordInit:
+  Description: 'Checks for redundant use of `keyword_init: true` in Struct.new.'
+  StyleGuide: '#no-keyword-init-true-struct-new'
+  Enabled: true
+  SafeAutoCorrect: true
```

Comment:
It is necessary to describe the reasons for being unsafe in the `@safety` section of the documentation.